controllers/topics.controllers.js

```diff
@@ -1,7 +1,12 @@
 const { selectTopics } = require('../models/topics.models');
+const endpointsData = require('../endpoints.json');
 
 exports.getTopics = (req, res, next) => {
   selectTopics().then((rows) => {
     res.send({ topics: rows });
   });
 };
+
+exports.getEndpoints = (req, res) => {
+  res.send({ endpoints: endpointsData });
+};
```

Comment:
Great work with requiring in the endpoints directly! Smart way of going about it - but you can take this even further and do the same in your test.